cypress/support/commands.js

```diff
@@ -13,3 +13,5 @@ Cypress.Commands.add('login', () => {
     body: users.admin,
   });
 });
+
+Cypress.Commands.add('getElement', element => cy.get('[data-test="' + element + '"]'));
```

Comment:
Maybe `getByTestId` to better convey the purpose?

Reply:
Sure. That was actually one of those moments I was staring at the screen and couldn't think of any better, although unsatisfied with `getElement`  :man_facepalming: